samples/scaleout/distributor-upgrade/sample.md

```diff
@@ -70,6 +70,10 @@ In Versions 6 and above there is a new API for enlisting with the distributor. T
 
 snippet:Enlisting
 
+The last parameter, `capacity` is the maximum number of messages in-flight between the distributor and the worker. The worker sends an acknowledgement (ACK) message (also called `ready` message) for each message processed. The distributor keeps track of these ACKs to ensure no worker is flooded with work that it can't process
```

Comment:
ie shouldnt this api be documented https://github.com/Particular/NServiceBus.Distributor.Msmq/blob/develop/src/NServiceBus.Distributor.MSMQ/IWorkerAvailabilityManager.cs#L20  ?